bbc/simorgh
# Support `atomicVersion` block on MediaArticlePage

Overall changes
======
- Adds support for the `atomicVersion` block type to MediaArticlePages
- This block type is used to render the 'live' stream on the likes of https://www.bbc.com/arabic/media-49522519
- **Consider moving this transform out of Simorgh and into the BFF**

Code changes
======
- Adds component check for `atomicVersion` block
- Adds converter function for `atomicVersion` block

Testing
======
1. Visit http://localhost:7080/persian/media-49522521?renderer_env=live
2. Confirm the video player loads and plays correctly

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
